packages/storefront/addon/components/spree-taxonomy.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+import layout from '../templates/components/spree-taxonomy';
+export default Ember.Component.extend({
+  layout: layout,
+  childTaxons: function(){
+    var taxons = this.get('taxonomy.taxons');
+    var parentTaxon = taxons.findBy('parentId', null);
+    var childTaxons = taxons.filter(function(taxon){
+      return taxon.get('parentId') === parentTaxon.get('id');
+    });
+    return childTaxons;
+  }.property('taxonomy.taxons')
```

Comment:
Can we please change this to the non-prototype syntax?

```
childTaxons: Ember.computed('taxonomy.taxons', function() {...
```

More info: http://reefpoints.dockyard.com/2015/03/22/tips-for-writing-ember-addons.html